src/encoded/static/components/rnaget.js

```diff
@@ -9,6 +9,43 @@ import QueryString from '../libs/query_string';
 import * as globals from './globals';
 import { TableItemCount } from './objectutils';
 import { FacetList, TextFilter } from './search';
+import { FetchedData, Param } from './fetched';
+
+
+const AutocompleteBox = (props) => {
+    const terms = props.suggestions['@graph'];
+    const { handleClick } = props;
+
+    /* eslint-disable jsx-a11y/no-noninteractive-element-interactions, jsx-a11y/no-noninteractive-tabindex, jsx-a11y/click-events-have-key-events, arrow-body-style */
```

Comment:
Why are you disabling eslint-events on a function that is not an if-statement?

Reply:
I modified an existing Autocomplete element we have. I didn't reuse it because it was way too complex for this very simple scenario. The original Autocomplete component also turns off these eslint rules.

The render ended up very simple and easy to read, but still breaking ESlint with rules such as:
`onClick={() => handleClick(term.text)}` not being allowed. But I don't see a simpler way to fix it the way we have it. As we are going to rewrite the interface, I just decided to move on with simplicity and turn off the rules just for these lines